src/main/java/net/minestom/server/entity/metadata/display/TextDisplayMeta.java

```diff
@@ -0,0 +1,93 @@
+package net.minestom.server.entity.metadata.display;
+
+import net.kyori.adventure.text.Component;
+import net.minestom.server.entity.Entity;
+import net.minestom.server.entity.Metadata;
+import org.jetbrains.annotations.NotNull;
+
+public class TextDisplayMeta extends AbstractDisplayMeta {
+    public static final byte OFFSET = AbstractDisplayMeta.MAX_OFFSET;
+    public static final byte MAX_OFFSET = OFFSET + 5;
+
+    private static final byte SHADOW = 1;
+    private static final byte SEE_THROUGH = 2;
+    private static final byte USE_DEFAULT_BACKGROUND = 4;
+    private static final byte ALIGN_LEFT = 8;
+    private static final byte ALIGN_RIGHT = 16;
+
+    public TextDisplayMeta(@NotNull Entity entity, @NotNull Metadata metadata) {
+        super(entity, metadata);
+    }
+
+    public @NotNull Component getText() {
+        return super.metadata.getIndex(OFFSET, Component.empty());
+    }
+
+    public void setText(@NotNull Component value) {
+        super.metadata.setIndex(OFFSET, Metadata.Chat(value));
+    }
+
+    public int getLineWidth() {
+        return super.metadata.getIndex(OFFSET + 1, 200);
```

Comment:
replace with line width

Reply:
the problem is the data type or the default value?

Reply:
I was mistaken, this is fine